webcompat/static/js/lib/issues.js

```diff
@@ -489,6 +490,21 @@ issues.MainView = Backbone.View.extend({
 
   handleKeyShortcuts: function() {
     Mousetrap.bind("mod+enter", _.bind(this.addNewComment, this));
+  },
+
+  // Auto expanding feature for comments
+  // See Line 473 in bugform.js 
```

Comment:
Just as a heads up, line numbers will change very quickly and become stale. A comment like "see autoExpand in bugform.js and keep it in sync with this" or something is good enough.